partner_firstname/models.py

```diff
@@ -38,12 +38,16 @@ class ResPartner(models.Model):
         required=False,
         store=True)
 
+    @api.model
+    def _get_computed_name(self, lastname, firstname):
+        """Write the 'name' field according to splitted data."""
+        return u" ".join((p for p in (lastname, firstname) if p))
```

Comment:
@pedrobaeza I think we can't change the behaviour of this module in this respect. 

`_get_computed_name` and `_get_inverse_name` are intented to be overridden to implement other schemes such as "name, firstname".

@adrienpeiffer I suggest you add a word in the README and the docstrings of `_get_computed_name` and `_get_inverse_name` to explain this.